src/Compilers/CSharp/Test/Emit3/Semantics/CollectionExpressionTests_WithElement_Extra.cs

```diff
@@ -0,0 +1,5828 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
These are the original tests in the dict-expr branch.  For sections that aren't supported yet (interfaces, create_methods, and with+dict) i have put things in if-def'ed regions.  we can then enable those regions in the PRs that follow this that add support.

Reply:
I only skimmed this file, since tests here were previously approved.